<launch>
  <arg name="name" default="$(env UAV_NAMESPACE)" />

  <!-- Pole AIC Inspection Node -->
  <node pkg="impedance_control" type="pole_aic_inspection" name="pole_aic_inspection" output="screen" ns="$(arg name)">
    <param name="name" value="$(arg name)" />
  </node>

  <!-- Reference Node -->
  <node pkg="impedance_control" type="reference_node" name="eference_node" output="screen" ns="$(arg name)">
  </node>

   <!-- Force-Trque sensor transformation publishe -->
  <node pkg="impedance_control" type="sensor2drone_node" name="sensor2drone_node" output="screen" ns="$(arg name)"/>

  <node pkg="impedance_control" type="force_filter_node" name="force_filter_node" output="screen" ns="$(arg name)"/>
</launch>
